src/command_modules/azure-cli-managementpartner/azure/cli/command_modules/managementpartner/_help.py

```diff
@@ -0,0 +1,51 @@
+﻿# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from knack.help_files import helps
+
+helps['managementpartner'] = """
+    type: group
+    short-summary: Manage Azure ManagementPartner.
+"""
+
+helps['managementpartner create'] = """
+    type: command
+    short-summary: Create new management partner
```

Comment:
Long summary text looks clearer than the short summary. Why not get rid of the long summary and use its text for the short one?